src/client/routes/Market/index.tsx

```diff
@@ -96,6 +102,13 @@ export const Market = () => {
 
   let ctaButtonText = userWallet ? `${t('market:banner.cta-borrower')}` : `${t('components.connect-button.connect')}`;
 
+  useEffect(() => {
+    if (!userWallet) {
```

Comment:
Plz move up towards rest of react hooks so we dont accidentally return early.

can u make it just if userWallet. It would be 3 instead of 4. might as well and if we want to add other things to useEffect then we dont want to return early

Reply:
This was just for testing. Removed.